README.md

```diff
@@ -819,6 +819,40 @@ public class YourCardPartsViewController: CardPartsViewController {
 }
 ```
 
+## Delegates
+Any view controller which is a subclass of CardPartsViewController supports  gesture delegate for long press on the view. Just need to conform your controller to CardPartsLongPressGestureRecognizerDelegate protocol.
+
+When the view is long pressed `didLongPress(_:)` will be called where you can custom handle the gesture.
+Example: Zoom in and Zoom out on gesture state begin/ended. 
+```swift
+    func didLongPress(_ gesture: UILongPressGestureRecognizer) -> Void
+```
+
+You can set the minimumPressDuration for your press to register as gesture began. The value is in seconds.
+`default is set to 1 second`.
+```swift
+    var minimumPressDuration: CFTimeInterval { get } // In seconds
+```
+
+Example:
+```swift
+extension MYOwnCardPartController: CardPartsLongGestureRecognizerDelegate {
+	func didLongPress(_ gesture: UILongPressGestureRecognizer) {
+		guard let v = gesture.view else { return }
```

Comment:
Where do you use `v` after unwrapping?

Reply:
Or are you just showing how you can get the gesture's view and do something??

Reply:
Just showing that could use v to animate :D